agents/plugins/mk_oracle

```diff
@@ -507,33 +507,106 @@ set_ora_version () {
+                  from v\$instance i
+                  join v\$con_sys_time_model s on s.stat_name in('DB time', 'DB CPU')
+                  join v\$containers vd on vd.con_id = s.con_id
+                  join v\$database d on d.cdb = 'YES'
+                  where vd.con_id <> 2
+                  union all
+                  select d.cdb, i.instance_name, s.stat_name, s.value, null
+                  from v\$instance i
+                  join v\$sys_time_model s on s.stat_name in('DB time', 'DB CPU')
+                  join v\$database d on d.cdb = 'NO'
+                  order by stat_name);
+
+              WITH iostat_file AS (
+                   SELECT
+                       con_id,
+                       filetype_name,
+                       SUM(large_read_reqs) large_read_reqs,
+                       SUM(large_read_servicetime) large_read_servicetime,
+                       SUM(large_write_reqs) large_write_reqs,
+                       SUM(large_write_servicetime) large_write_servicetime,
+                       SUM(small_read_reqs) small_read_reqs,
+                       SUM(small_read_servicetime) small_read_servicetime,
+                       SUM(small_sync_read_reqs) small_sync_read_reqs,
+                       SUM(small_write_reqs) small_write_reqs,
+                       SUM(small_write_servicetime) small_write_servicetime,
+                       SUM(small_read_megabytes*1024*1024) small_read_bytes,
+                       SUM(large_read_megabytes*1024*1024) large_read_bytes,
+                       SUM(small_write_megabytes*1024*1024) small_write_bytes,
+                       SUM(large_write_megabytes*1024*1024) large_write_bytes
+                   FROM v\$iostat_file
+                   GROUP BY con_id, filetype_name
+               )
+               SELECT
+                   upper(DECODE(d.cdb,'NO',i.instance_name
+                                     ,i.instance_name || '.' || vd.name) )
+                   || '|iostat_file'
+                   || '|' || filetype_name
+                   || '|' || small_read_reqs
+                   || '|' || large_read_reqs
+                   || '|' || small_write_reqs
+                   || '|' || large_write_reqs
+                   || '|' || small_read_servicetime
+                   || '|' || large_read_servicetime
+                   || '|' || small_write_servicetime
+                   || '|' || large_write_servicetime
+                   || '|' || small_read_bytes
+                   || '|' || large_read_bytes
+                   || '|' || small_write_bytes
+                   || '|' || large_write_bytes
+               FROM
+                   iostat_file io
+                   JOIN v\$containers vd ON io.con_id = vd.con_id
+                   JOIN v\$instance i ON 1 = 1
+                   JOIN v\$database d ON 1 = 1
+               ORDER BY vd.con_id, io.filetype_name;
+              select upper(decode(con_id, null, instance_name
```

Comment:
This part leads to an output like this if the database is a single (no multitenant):
`MYSID.|sys_wait_class|Administrative|2|510|2|510`
Is the trailing dot intended?